ministryofjustice/laa-claim-non-standard-magistrate-fee-backend
# CRM457-1108: Enable submission to app store

## Description of change
Enable submission to app store

[Link to relevant ticket](https://dsdmoj.atlassian.net/browse/CRM457-1108)

## How to manually test the feature
Submit on branch then check app store and or case worker app for the relevant application.
